README.md

```diff
@@ -29,7 +33,34 @@ Empty registers are not shown by default.
 
 Use the <kbd>Up</kbd> and <kbd>Down</kbd> or <kbd>Ctrl</kbd><kbd>P</kbd> and <kbd>Ctrl</kbd><kbd>N</kbd> or <kbd>Ctrl</kbd><kbd>J</kbd> and <kbd>Ctrl</kbd><kbd>K</kbd> keys to select the register you want to use and press <kbd>Enter</kbd> to apply it, or type the register you want to apply, which is one of the following:
 
-<kbd>"</kbd> <kbd>0</kbd>-<kbd>9</kbd> <kbd>a</kbd>-<kbd>z</kbd> <kbd>:</kbd> <kbd>.</kbd> <kbd>%</kbd> <kbd>#</kbd> <kbd>=</kbd> <kbd>\*</kbd> <kbd>+</kbd> <kbd>\_</kbd> <kbd>/</kbd>
+<kbd>"</kbd> <kbd>0</kbd>–<kbd>9</kbd> <kbd>a</kbd>–<kbd>z</kbd> <kbd>:</kbd> <kbd>.</kbd> <kbd>%</kbd> <kbd>#</kbd> <kbd>=</kbd> <kbd>\*</kbd> <kbd>+</kbd> <kbd>\_</kbd> <kbd>/</kbd>
+
+## Example Workflow
+
+### With `registers.nvim`:
+
+- Copy item to system clipboard or vim register (this can also be achieved using `registers.nvim`);
+- In normal mode, type the <kbd>"</kbd> key. The `registers.nvim` pop-up will appear.
+- You can now use the arrow keys (or other navigation keys) to move to a specific item (representing an item in the vim register, i.e., what you see with `:reg`) in the list being displayed. Pressing enter on this item will ‘select’ it.
+- Alternatively, you can just type the highlighted character being displayed in the left margin of the pop-up, next to the register item you want to select. This resembles a workflow without `registers.nvim`, except that you get the visual feedback and confirmation of what is inside the register beforehand. It is also useful for remembering which register you placed something in ;) .
+    + I.e., type <kbd>"</kbd><kbd>+</kbd> to select from the system clipboard, **or** use the arrow keys to navigate to <kbd>+</kbd> and hit enter)
+- Once you have selected a register item, you can proceed to perform your desired action (e.g., yank, paste, etc.). To do this, simply use the usual keys: <kbd>y</kbd>, <kbd>p</kbd>, etc.
+
+One can also call the `registers.nvim` pop-up through other means, not just <kbd>"</kbd>. For example, in insert mode, one can use <kbd>Ctrl</kbd><kbd>R</kbd> (this is default vim behaviour). If one does not want to use these key-binds (or bind your own keys), one can use the `:Registers` command.
+
+### Without `registers.nvim`:
+
+- Copy item to system clipboard or vim register;
+- Use the `:reg` command to view registers. Make sure to remember the `Name` of the register item you want to use for future reference.
+- In normal mode, type <kbd>"</kbd>. You will get no visual feedback.
+- Now type the `Name` of the register item you want to select, as listed in the output of `:reg`.
+- Now perform the desired action (usually either yanking, <kbd>y</kbd>, or pasting, <kbd>p</kbd>)
+
+### Using `registers.nvim` is definitely more aesthetically pleasing and probably makes registers easier for beginners to understand—but is it actually better for experienced neovim users, too?
```

Comment:
```suggestion
### Using `registers.nvim` is definitely more aesthetically pleasing and probably makes registers easier for beginners to understand—but is it actually better for experienced Neovim users, too?
```

Isn't this text also a bit long for a header?

Reply:
Yeah, it is. Not sure what else to do with it though… if you have any suggestions feel free to change it.